lib/usbmux/usbmux.js

```diff
@@ -102,7 +103,7 @@ class Usbmux {
     if (data.payload.DeviceList) {
       return data.payload.DeviceList;
     } else {
-      throw new Error(data);
+      throw new Error(`Unexpected data: ${JSON.stringify(data)}`);
```

Comment:
I should do this to the other places I throw errors as well

Reply:
Yeah. Otherwise the error comes out as `[Object object]`. Because JavaScript.

https://www.destroyallsoftware.com/talks/wat